src/modules/decide.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+var random = require('random-js')()
+
+module.exports = {
+  names: ['decide'],
+  onMessage: function (bot, user, userID, channelID, message, rawEvent) {
+    var messages = message.split(' ')
+    messages.shift()
+    messages.splice(1, 1)
```

Comment:
consider maybe using random-js' `pick` function to make this a little more concise. example below
```js
  onMessage: function (bot, user, userID, channelID, request, rawEvent) {
    const message = random.pick(request.split(' ').slice(1).filter(x => x !== 'or'))
    bot.sendMessage({
      to: channelID,
      message
    })
  }
```